comp/api/api/apiimpl/api_test.go

```diff
@@ -29,7 +29,8 @@ import (
 	"github.com/DataDog/datadog-agent/comp/core/tagger"
 	"github.com/DataDog/datadog-agent/comp/core/tagger/taggerimpl"
 	"github.com/DataDog/datadog-agent/comp/core/workloadmeta"
-	"github.com/DataDog/datadog-agent/comp/dogstatsd/replay"
+	replaydef "github.com/DataDog/datadog-agent/comp/dogstatsd/replay/def"
```

Comment:
Should we keep his as `replay` rather than `replaydef` for consistency with the rest of the files?